cdap-api/src/main/java/co/cask/cdap/api/artifact/PluginConfigurer.java

```diff
@@ -25,7 +26,7 @@
  * This interface provides methods to register plugin usage in a CDAP Program. The registered plugins will be
  * available at the runtime of the Program.
  */
-public interface PluginConfigurer {
+public interface PluginConfigurer extends DatasetConfigurer {
```

Comment:
Currently we send the Plugin information used in a specific program through its ProgramSpecification. So if we want to support adding Plugins in the Application, then we need to pass that info through ApplicationSpecification. Should we support that? It is easier for streams/datasets since it should be unique across the app and programs. But for plugins, we want to support local plugins and global plugins?